### PR TITLE
chore(doc): update help for workflows

### PR DESCRIPTION
**Pull request by benjaminParisel**
* just push a slack notification when release is done

Covers[UID-314](https://bonitasoft.atlassian.net/browse/UID-314)

### CODE REVIEW

File: .github/workflows/notify-studio-pr.yml [hunk 1]
```diff
@@ -1,15 +1,5 @@
 name: Bump version
```
Comment (jeromecambon) [inline .github/workflows/notify-studio-pr.yml:1]: Update name

File: .github/workflows/notify-studio-pr.yml [hunk 2]
```diff
@@ -20,17 +10,21 @@ jobs:
...
+      with:
+        status: custom
+        fields: workflow,job,commit,repo,ref
+        custom_payload: |
+          {
+            username: 'action-slack',
+            icon_emoji: ':octocat:',
+            attachments: [{
+              color: 'good',
+              text: `UI-Designer release ${{ github.event.client_payload.releaseVersion }} success. Next step: Open PR in bonita-studio on branch ${{github.event.client_payload.baseBranch}} repository and close nexus release`,
```
Comment (jeromecambon) [inline .github/workflows/notify-studio-pr.yml:26]: If you could remain the file to update in Studio, and provide the link for Nexus, il would be even better!

File: .github/help.adoc [hunk 3]
```diff
@@ -8,10 +8,12 @@ On each push/pull_request on master and dev branch, *build* is trigger
...
-|Workflow name|description
-|Release| Create github release and could optionally call `Publish binaries` workflow
-|Publish binaries|Push binaries on repository (maven central), as asset on github release and Could optionally call `Open Studio PR` workflow
-|Open Studio PR|Open a PR on studio repository with new uid release version
+|Workflow name|description|Start by
+|Build| Build project on each PR or commit on master/dev| automatically
+|Release| Create github release and could optionally call `Publish binaries` workflow| manually
+|Publish binaries|Push binaries on repository (maven central), as asset on github release and Could optionally call `Open Studio PR` workflow| manually or automatically
+|Crowdin download| Download from crowdin all translation and open a PR|automatically
+|Commit Message check| Check commit message when a PR is open|automatically
```
Comment (jeromecambon) [inline .github/help.adoc:16]: Add Notify Studio PR

**Review by jeromecambon — CHANGES_REQUESTED** [review]

File: .github/workflows/notify-studio-pr.yml (hunk 2 above)
Comment (benjaminParisel) [reply .github/workflows/notify-studio-pr.yml:26]: I need to update it, we don't need to go on nexus now ;)

**Review by jeromecambon — APPROVED** [review]
LGTM, but conflict to be resoved on the help file

**Pull request merged**